.pipelines/npm/npm-conformance-tests.yaml

```diff
@@ -463,7 +463,169 @@ jobs:
         name: cyclonus
         displayName: "Run Cyclonus Test"
         failOnStderr: false
+        condition: succeeded()
+
+      - bash: |
```

Comment:
diff says we're adding this block, but really we're just copying it from here to below the scale test